dlp/src/main/java/dlp/snippets/DeIdentifyWithMasking.java

```diff
@@ -0,0 +1,88 @@
+package dlp.snippets;
+
+// [START dlp_deidentify_masking]
+
+import com.google.cloud.dlp.v2.DlpServiceClient;
+import com.google.privacy.dlp.v2.CharacterMaskConfig;
+import com.google.privacy.dlp.v2.ContentItem;
+import com.google.privacy.dlp.v2.DeidentifyConfig;
+import com.google.privacy.dlp.v2.DeidentifyContentRequest;
+import com.google.privacy.dlp.v2.DeidentifyContentResponse;
+import com.google.privacy.dlp.v2.InfoType;
+import com.google.privacy.dlp.v2.InfoTypeTransformations;
+import com.google.privacy.dlp.v2.InfoTypeTransformations.InfoTypeTransformation;
+import com.google.privacy.dlp.v2.InspectConfig;
+import com.google.privacy.dlp.v2.PrimitiveTransformation;
+import com.google.privacy.dlp.v2.ProjectName;
+import com.google.privacy.dlp.v2.ReplaceWithInfoTypeConfig;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class DeIdentifyWithMasking {
+
+  public static void deIdentifyWithMasking() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String textToDeIdentify = "My SSN is 372819127";
+    deIdentifyWithMasking(projectId, textToDeIdentify);
+  }
+
+  public static void deIdentifyWithMasking(String projectId, String textToDeIdentify)
+      throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DlpServiceClient dlp = DlpServiceClient.create()) {
+
+      // Specify what content you want the service to DeIdentify
+      ContentItem contentItem = ContentItem.newBuilder().setValue(textToDeIdentify).build();
+
+      // Specify the type of info the inspection will look for.
+      // See https://cloud.google.com/dlp/docs/infotypes-reference for complete list of info types
+      List<InfoType> infoTypes =
+          Stream.of("US_SOCIAL_SECURITY_NUMBER")
```

Comment:
Why use this stream/list creation instead of adding the single info type like in the other examples?

Reply:
Done.